### DISCUSSION TITLE
Permission Error when trying to start the Odoo container on a Mac

### DISCUSSION THREAD

**Discussion by piqoni** [Tecnativa/doodba#401] [Q&A]
Hi there, first time trying to setup doodba on a mac (OS: Big Sur). Got this permission error but its unclear where it is coming from. Any suggestions? 

```
➜  docker git:(master) ✗ sudo docker-compose up odoo
WARNING: The CPSMS_APIKEY variable is not set. Defaulting to a blank string.
Building with native build. Learn about native build in Compose here: https://docs.docker.com/go/compose-native-build/
docker_smtp_1 is up-to-date
docker_db_1 is up-to-date
Starting docker_wdb_1 ... done
Starting docker_odoo_1 ... done
Attaching to docker_odoo_1
odoo_1                    | Traceback (most recent call last):
odoo_1                    |   File "/opt/odoo/common/entrypoint", line 30, in <module>
odoo_1                    |     subprocess.check_call(command)
odoo_1                    |   File "/usr/local/lib/python3.5/subprocess.py", line 266, in check_call
odoo_1                    |     retcode = call(*popenargs, **kwargs)
odoo_1                    |   File "/usr/local/lib/python3.5/subprocess.py", line 247, in call
odoo_1                    |     with Popen(*popenargs, **kwargs) as p:
odoo_1                    |   File "/usr/local/lib/python3.5/subprocess.py", line 676, in __init__
odoo_1                    |     restore_signals, start_new_session)
odoo_1                    |   File "/usr/local/lib/python3.5/subprocess.py", line 1289, in _execute_child
odoo_1                    |     raise child_exception_type(errno_num, err_msg)
odoo_1                    | PermissionError: [Errno 13] Permission denied
docker_odoo_1 exited with code 1
```

**Comment by piqoni** [comment]
Any hint so I can narrow it down somewhere would be useful at this point 🙏

**Reply by joao-p-marques** [reply]
Hi @piqoni 

Can you take a look at https://github.com/Tecnativa/doodba-copier-template/issues/164 for reference? It seems like you stumbled upon the same issue.

**Reply by piqoni** [reply]
Wow thank you so much. The permission error went away after deleting the .empty file. THANK YOU!!

I got another error but will see what I can do or will open a separate discussion thread if no results. Worse case, I disable Inotify somehow. 
(AttributeError: 'Inotify' object has no attribute '_Inotify__inotify_fd')

**Reply by joao-p-marques** [reply]
Nice!

As for the other Inotify error, that is also an old recurrent question. Check if this helps: https://code.visualstudio.com/docs/setup/linux#_visual-studio-code-is-unable-to-watch-for-file-changes-in-this-large-workspace-error-enospc

**Reply by piqoni** [reply]
Thanks, for others that might end up in this thread. I disabled them completely in server.py
![image](https://user-images.githubusercontent.com/3144671/113991746-0c3d8d00-9853-11eb-8f76-379b149cd45c.png)

**Reply by HviorForgeFlow** [reply]
It should be easier to drop reload on --dev configuration. You should have plenty of code to get this error. I had same problem few months ago. Not related with MAC

**Reply by piqoni** [reply]
I agree, but was not sure where that setting is located in the Doodba setup and wanted to move forward. Let me know if someone knows the best practices on that :)

**Reply by pedrobaeza** [reply]
As Héctor has said, just remove reload on devel.yaml. Your patch is very ugly.

**Reply by piqoni** [reply]
Absolutely! Thanks Pedro, didnt know it was in devel.yaml ! 🙌
